docs/api_v2/api_v2_reference.apib

```diff
@@ -4018,53 +4034,55 @@ The user account is identified by the provided authentication token.
+- email:                 `bob@example.com`                  (string, required)
+- proxy                                                     (object, required)
+    - password:          `ad78knd9Jkjd86`                   (string, required)
+    - groups                                                (array, required)
+- plan                                                      (object, required)
+    - id:                               `Personal`          (string, required)
+    - description:                      `Cost-effective plan for freelancers, developers and students.` (string, required)
+    - isEnabled:                        true                (boolean, required)
+    - monthlyBasePriceUsd:              49                  (number, required)
+    - monthlyUsageCreditsUsd:           49                  (number, required)
+    - usageDiscountPercent:             0                   (number, required)
+    - enabledPlatformFeatures                               (array, required)
+        - `ACTORS`                                          (array, required)
+        - `STORAGE`                                         (array, required)
+        - `PROXY_SERPS`                                     (array, required)
+        - `SCHEDULER`                                       (array, required)
+        - `WEBHOOKS`                                        (array, required)
+    - maxMonthlyUsageUsd:                9999               (number, required)
+    - maxActorMemoryGbytes:              32                 (number, required)
+    - maxMonthlyActorComputeUnits:       1000               (number, required)
+    - maxMonthlyResidentialProxyGbytes:  10                 (number, required)
+    - maxMonthlyProxySerps:              30000              (number, required)
+    - maxMonthlyExternalDataTransferGbytes: 1000            (number, required)
+    - maxActorCount:                    100                 (number, required)
+    - maxActorTaskCount:                1000                (number, required)
+    - dataRetentionDays:                14                  (number, required)
+    - availableProxyGroups                                  (object, required)
+      - SOMEGROUP:                      20                  (number, required)
+      - ANOTHERGROUP:                   200                 (number, required)
+    - teamAccountSeatCount:             1                   (number, required)
+    - supportLevel:                     `COMMUNITY`         (string, required)
+    - availableAddOns                                       (array, required)
+
+## UserUsage (object)
+- usageCycle                                        (object, required)
+    - startAt:      `2020-11-20T00:00:00.000Z`      (string, required)
+    - endAt:        `2020-12-19T23:59:59.999Z`      (string, required)
+- usage                                             (object, required)
+    - TOTAL_USAGE_USD:      0.00023407795744696     (number, nullable)
```

Comment:
I am not sure about this being snake cased capitalized but it's how we have it everywhere internally.

Reply:
Hm, it doesn't look good. When it was internal it didn't matter too much, but now it will be like this forever...

Can we perhaps rename `usage` to `platformUsage` (for consistency with UI) and move the `TOTAL_USAGE_USD` to `totalPlatformUsageUsd` ?